templates/compositions/ec-component-organisation-teaser/organisation-teaser.html.twig

```diff
@@ -0,0 +1,117 @@
+{% spaceless %}
+
+{#
+/**
+ * @file
+ * Theme implementation for ECL Organisation Teaser.
+ *
+ * Available variables:
+ * - "name" (string) (default: ''): name of the organisation.
+ * - "acronym" (string) (default: ''): acronym of the organisation.
+ * - "logo" (ImageValueObject) (default: {}): logo of the organisation.
+ * - "organisation_details" (array) (default: []): list of fields.
+ * - "contact_page_url" (string) (default: ''): email adress or contact page to contact the organisation.
+ * - "contact_page" (array) (default: []): processed organisation contact page url.
+ */
+#}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-content-item ecl-u-d-flex ecl-u-pb-m' %}
+{% set _extra_attributes = '' %}
+{% set _name = name|default('') %}
+{% set _acronym = acronym|default('') %}
+{% set _logo = logo|default({}) %}
+{% set _organisation_details = organisation_details|default([]) %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
```

Comment:
Same here, we don't have to provide extra classes, or extra attributes. Let's remove all this.